demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -78,6 +78,8 @@ class BaseContent(ABC, BaseModel, metaclass=BaseContentMetaclass):
     content_type: ClassVar[ContentType] = Field(include=True)
     node_id: str
     marketplaces: List[MarketplaceVersions] = list(MarketplaceVersions)
+    not_in_repository: bool = False
+    name: str = ""
```

Comment:
Were `name` attributes only in `ContentItem` before?

Reply:
No. The name attribute was in UnknownContent as well, so in order to avoid mypy error, I added it to BaseContent.

Reply:
Can we now remove it from objects inheriting from BaseContent?